three-cards-divination/pom.xml

```diff
@@ -16,5 +16,13 @@
         <maven.compiler.target>21</maven.compiler.target>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     </properties>
+    <dependencies>
+        <dependency>
+            <groupId>org.mockito</groupId>
+            <artifactId>mockito-core</artifactId>
+            <version>5.7.0</version>
+            <scope>test</scope>
+        </dependency>
+    </dependencies>
```

Comment:
**Suggestion:** It's better to manage the version of `mockito-core` in the parent pom.xml file. This way, all modules will use the same version of the library, ensuring consistency across the project. [best practice]
```suggestion
    <dependencies>
        <dependency>
            <groupId>org.mockito</groupId>
            <artifactId>mockito-core</artifactId>
            <scope>test</scope>
        </dependency>
    </dependencies>
```